far/FarEng.hlf.m4

```diff
@@ -6343,92 +6343,91 @@ in HKCU hive is ignored.
 
 @Editor.KeepEditorEOL
 $ #far:config Editor.KeepEditorEOL#
- Параметр "Editor.KeepEditorEOL" позволяет управлять типом перевода строк для текста
-вставляемого в редактор из буфера обмена.
+ This Boolean parameter controls how line breaks within the text on the
+clipboard are pasted into the edited file.
 
- false - ^<wrap>текст вставляется без изменений, т.е. берётся тип перевода строк источника (при этом можно получить текст
-с разными типами перевода строк).
+ False - ^<wrap>Line breaks in the pasted text are preserved. After the
+paste operation, line breaks in the edited file may have different styles.
+ True  - If the file is not empty, line breaks in the pasted text are
+changed to match the line break style of the edited file. If the file
+is empty, line breaks are not changed; this parameter has no effect.
 
- true  - ^<wrap>если файл не пуст, сохраняется текущий тип перевода строк редактируемого файла,
-в противном случае параметр ни на что не влияет, т.е. будет использован стиль переводов строк оригинала.
+ Default value: True (match line break style of the edited file).
 
- По умолчанию значение = true
-
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @Editor.AddUnicodeBOM
 $ #far:config Editor.AddUnicodeBOM#
- Параметр "Editor.AddUnicodeBOM" указывает надо ли добавлять BOM (Byte Order Mark) в начало создаваемых
-редактором файлов в юникодной кодировке (1200, 1201, 65001).
+ This Boolean parameter specifies whether Byte Order Mark (BOM) is added
+at the beginning of the files created by the Editor and saved in
+a UNICODE encoding (code pages 1200, 1201, 65001).
 
- false - ^<wrap>BOM не добавляется.
+ False - ^<wrap>BOM is not added.
+ True  - BOM is added.
 
- true  - ^<wrap>BOM добавляется.
+ Default value: True (BOM is added).
 
- По умолчанию значение = true
-
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @Editor.NewFileUnixEOL
 $ #far:config Editor.NewFileUnixEOL#
- Параметр "Editor.NewFileUnixEOL" позволяет задавать умолчательные символы конца строки в редакторе
-для вновь создаваемых файлов.
-
- false - ^<wrap>Во вновь создаваемых файлах строки заканчиваются символами #<CR><LF># - (как в Windows).
+ This Boolean parameter specifies line break style in the files created
```

Comment:
[Line endings](https://www.microsoft.com/en-us/language/Search?&searchTerm=line%20endings&langID=635&Source=true&productid=0) maybe?

Reply:
"Line ending" appears in just one place in FarEng.hlf.m4 file, while "line break" is in at least 18 places (likely more). This is why I choose the latter term. I can live with either. If you have strong preference, please let me know, I'll change all places to be consistent.

Reply:
I don't have a strong preference, let's keep it consistent with the rest of the file.

Reply:
Changed to `line break` (plus minor cleanup).